src/main/scala/higherkindness/rules_scala/workers/common/FileUtil.scala

```diff
@@ -59,6 +59,29 @@ object FileUtil {
     path.getFileName().toString().stripPrefix("header_").stripPrefix("processed_")
   }
 
+  /**
+   * Given a Bazel path to the bazel output directory, return a path excluding the bazel configuration specific parts of
+   * the path. This is analogous to Bazel's File.short_path function. Fair warning: this function is super good enough,
+   * but is likely not perfect.
+   */
+  def bazelShortPath(path: Path, replaceExternal: Boolean = true): Path = {
```

Comment:
I put this in a function because I use this code in the path mapping PR for the deps checker and want to be able to reuse it in that PR

Reply:
Thanks for doing that.